packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -186,6 +186,11 @@ export const ClayCardWithInfo = ({
 							: 'primary'
 					}
 					position="bottom-left"
+					title={
+						stickerProps.content
+							? stickerProps.content
+							: stickerSymbol
```

Comment:
@matuzalemsteles 
For this, I changed the line to just `title={stickerProps.title}`.

> add it to the type as well to allow passing the title in the stickerProps. In this case we don't need to force some value for the title.

For this portion of your comment, how/where should this change be made?

Reply:
It was supposed to update the type https://github.com/liferay/clay/blob/40e3d5c8f70bbcecf942314d1b6bd232e97c6b2b/packages/clay-card/src/CardWithInfo.tsx#L103, but as it already extends `IClayStickerProps‎` it would no longer need to add the type to `title` which already has that type defined.

Reply:
So just changing this to `title={stickerProps.title}` is sufficient?